README.md

```diff
@@ -0,0 +1,21 @@
+# gamesgames
+
+A Tic Tac Toe Console Application
+
+## Setup
+
+New users can begin by installing npm.
```

Comment:
maybe mention how to install npm, or link to something that explains it

Reply:
[Added link to their instructions.](https://github.com/saralein/tic-tac-toe_npm/pull/1/commits/a953b18d98909591146fc93f18b79af3838485df#diff-04c6e90faac2675aa89e2176d2eec7d8R7).